src/App.jsx

```diff
@@ -14,33 +15,83 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+const SORT_LENGTH = 'length';
+const SORT_ALPHABETICALLY = 'alphabetically';
 
-      <button type="button" className="button is-success is-light">
-        Sort by length
-      </button>
+const getSortedGood = (goods, sort, reverse) => {
+  const newGoods = [...goods];
 
-      <button type="button" className="button is-warning is-light">
-        Reverse
-      </button>
+  if (sort) {
+    newGoods.sort((goodOne, goodTwo) => {
+      switch (sort) {
+        case 'length':
+          return goodOne.length - goodTwo.length;
+        case 'alphabetically':
```

Comment:
Consider using the constants `SORT_LENGTH` and `SORT_ALPHABETICALLY` instead of hardcoded strings 'length' and 'alphabetically'. This will make the code more maintainable and reduce the risk of typos.